dm/tests/all_mode/run.sh

```diff
@@ -642,6 +642,83 @@ function run() {
 
 	run_sql_both_source "SET @@GLOBAL.SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION'"
 	run_sql_both_source "SET @@global.time_zone = 'SYSTEM';"
+
+	test_source_and_target_with_empty_gtid
+}
+
+function prepare_test_empty_gtid() {
+	cleanup_process
+
+	# clean test dir
+	rm -rf $WORK_DIR
+	mkdir $WORK_DIR
+
+	# kill the old tidb
+	pkill -hup tidb-server 2>/dev/null || true
+	wait_process_exit tidb-server
+
+	# restart tidb
+	run_tidb_server 4000 $TIDB_PASSWORD
```

Comment:
these actions are not needed? We can reuse the old DM cluster after stop task

Reply:
ok, I've deleted them